src/easyscore.ts

```diff
@@ -1,6 +1,7 @@
 // [VexFlow](https://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 // MIT License
 
+import { PartialBeamDirection } from '.';
```

Comment:
`from '.'` should be `from './beam`

Reply:
Good catch -- that was a VS Code autocomplete import, I'll watch for that in the future.

Reply:
I think at some point I'll suggest a VSCode workspace settings change, so that it will prefer to auto import files with the .js extension, which is more explicit and also supported by TypeScript. By default it opts for the shortest import, and currently importing from '.' can break our ESM build.